test/irmin-pack/migration.ml

```diff
@@ -218,10 +218,88 @@ module Make_layered =
     (Hash)
 module Test_layered_store = Test (Make_layered) (Config_layered_store)
 
+module Test_corrupted_stores = struct
```

Comment:
Added the tests along side the migration ones, because they are similar. We should probably rename the file into `test_existing_stores` (I'm not inspired)?

Reply:
`test_existing_stores` is better than anything I can come up with :+1: